src/wallet3/transaction_scanner.cpp

```diff
@@ -10,7 +10,12 @@
 namespace wallet
 {
   namespace log = oxen::log;
-  static auto logcat = log::Cat("wallet.wallet3");
+  static auto logcat = log::Cat("wallet");
+
+  bool is_coinbase(const BlockTX& tx)
```

Comment:
is this not / should this not be a method on cryptonote::transaction?

Reply:
Damn, it absolutely is defined already in `cryptonote_basic_impl`